tsconfig.json

```diff
@@ -11,10 +11,7 @@
     "jsx": "react",
     "module": "ESNext",
     "allowJs": true,
-    "declaration": true,
-    "declarationDir": "types",
     "sourceMap": true,
-    "outDir": "dist",
```

Comment:
Doing this in the separate:

```
{
		input: "src/index.ts",
		output: [{ file: "dist/index.d.ts", format: "esm" }],
		plugins: [dts()],
		external: [/\.css$/, /\.stories.tsx$/, /\.previewData.ts$/]
	}
```

otherwise Rollup and Typescript and the fact that there were 2 outputs which are now directories really confused each other.